aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,346 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  List,
+  RadioButtonGroup,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextArea,
+  TextInput,
+} from 'grommet';
+import {
+  Checkmark,
+  FormClose,
+  FormNextLink,
+  FormPreviousLink,
+} from 'grommet-icons';
+
+const defaultFormValues = {
+  'text-input-validation': '',
+  'radio-button-group-validation': '',
+  select: '',
+  checkboxgroup: '',
+  'text-area': '',
+};
+
+const stepOneValidate = values => {
+  const emailRegex = RegExp(
+    '[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+',
+  );
+  const emailValid = emailRegex.test(values['text-input-validation']);
+
+  return {
+    email: emailValid ? '' : 'Invalid email address.',
+    isValid: !!emailValid,
+  };
+};
+
+const StepOne = ({ active, setActive, formValues }) => {
+  const [error, setError] = useState({
+    email: '',
+    radiobuttongroup: '',
+  });
+
+  return (
+    <>
+      <Box margin={{ bottom: 'large' }}>
+        <FormField
+          label="Email"
+          htmlFor="text-input-validation"
+          name="text-input-validation"
+          error={error.email}
+          onChange={() => setError(stepOneValidate(formValues))}
+        >
+          <TextInput
+            placeholder="jane.smith@hpe.com"
+            id="text-input-validation"
+            name="text-input-validation"
+            type="email"
+          />
+        </FormField>
+        <FormField
+          help="Click on the ‘eye’ icon for the description text field to hide."
+          htmlFor="radio-button-group-validation"
+          label="RadioButtonGroup"
+          name="radio-button-group-validation"
+        >
+          <RadioButtonGroup
```

Comment:
For some reason, in full-screen mode, I am not able to make a selection for the radiobuttongroup input. May be unrelated to this PR, but noting for investigation.

Reply:
Good note, I think this is separate from this PR but we should file as an issue.

Reply:
Noticing this is happening even in just the radiobuttongroup page when you open to full screen. Tracking this issue here: #1031